test/app_test_case.py

```diff
@@ -1,17 +1,26 @@
 from abc import abstractmethod, ABCMeta
+from base64 import b64encode
+import json
+import math
 import os
 import sys
 import time
 import logging
+from subprocess import call
 import unittest
+import uuid
 
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+from jwt import encode as jwt_encode
 import requests
 from threading import Thread
+from chalice import Response
 # noinspection PyPackageRequirements
 from chalice.local import LocalDevServer
 # noinspection PyPackageRequirements
 from chalice.config import Config as ChaliceConfig
-from azul import config
+from azul import config as azul_config
```

Comment:
I want this to be`config` consistently across the code base. If some other name conflicts with that, I want that other thing to be renamed.

Reply:
ok

Reply:
The alias has been removed.